scyllacloud/hostdialer.go

```diff
@@ -96,8 +96,18 @@ func (s *SniHostDialer) dialInitialContactPoint(ctx context.Context, host *gocql
 		return nil, fmt.Errorf("can't get root CA from configuration: %w", err)
 	}
 
-	return s.connect(ctx, s.dialer, host.HostnameAndPort(), &tls.Config{
-		ServerName:         host.Hostname(),
+	contextConf, ok := s.connConfig.Contexts[s.connConfig.CurrentContext]
+	if !ok {
+		return nil, fmt.Errorf("currentContext in connection config points to not existing context")
```

Comment:
```suggestion
		return nil, fmt.Errorf("context %q does not exist", s.connConfig.CurrentContext)
```
also reminds me we should check the context isn't empty

Reply:
done